docs/workflows.md

```diff
@@ -367,14 +364,11 @@ import { createConsoleReader } from "examples/helpers/io.js";
 import { OpenMeteoTool } from "bee-agent-framework/tools/weather/openMeteo";
 import { WikipediaTool } from "bee-agent-framework/tools/search/wikipedia";
 import { AgentWorkflow } from "bee-agent-framework/experimental/workflows/agent";
-import { BaseMessage, Role } from "bee-agent-framework/llms/primitives/message";
-import { WatsonXChatLLM } from "bee-agent-framework/adapters/watsonx/chat";
+import { UserMessage } from "bee-agent-framework/backend/message";
+import { WatsonXChatModel } from "bee-agent-framework/adapters/watsonx/backend/chat";
```

Comment:
I don't remember the rules around this, but there's a chance this might need to be `WatsonxChatModel` (lowercase x)

Reply:
👍🏻 updated